aea/connections/p2p/stub/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
```

Comment:
@lrahmani the folder structure in the AEA framework needs to be strictly adhered to for it to work. Hence, you need to create  a folder `stub_p2p` in the `connections dir which contains the `connection.py` and connection.yaml` files.

Reply:
Also, we only allow for the default stub connection here. This package needs to move to `packages` dir so the user imports it. Will explain.